ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/project.pbxproj

```diff
@@ -811,10 +811,10 @@
 					"$(OTHER_LDFLAGS)",
 					"-ObjC",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.microsoft.OfficeUIFabricDemo-df";
+				PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.OfficeUIFabricDemo;
 				PRODUCT_NAME = FluentUI.Demo;
 				PROVISIONING_PROFILE = "63d62159-2691-4b44-9553-b668cc1746c1";
-				PROVISIONING_PROFILE_SPECIFIER = "Office Fabric Demo Dogfood development";
```

Comment:
I think we can just set the "Office Fabric Demo Dogfood development" for the dogfood and it's fine to return the other ones to the previous AppID.

Reply:
Sounds good! That's what this PR does, so let me know if there are any additional changes needed.

Reply:
Ah I see. On it!